src/features/goblins/bank/lib/bankUtils.ts

```diff
@@ -62,7 +63,10 @@ export function canWithdraw({ item, game }: CanWithdrawArgs) {
   }
 
   // Placed items
-  if (item in game.collectibles) {
+  if (
+    item in game.collectibles &&
+    game.collectibles[item as CollectibleName]?.length
```

Comment:
Add in the check for keys with an empty array value currently. This will avoid a migration.